formats/statistical_data_set/frontend/examples/statistical_data_set_withdrawn.json

```diff
@@ -0,0 +1,57 @@
+{
+  "base_path": "/government/statistical-data-sets/tsgb10-withdrawn",
```

Comment:
I'd prefer a real withdrawn example. This helps with:
* like by like comparisons when building or iterating the frontend
* confirming or refuting assumptions made when building the frontend for the first example